src/playlist-selectors.js

```diff
@@ -356,3 +357,25 @@ export const minRebufferMaxBandwidthSelector = function(settings) {
 
   return rebufferingEstimates[0] || null;
 };
+
+/**
+ * Chooses the appropriate media playlist, which in this case is the lowest bitrate
+ * one with video.  If no renditions with video exist, return the lowest audio rendition.
+ *
```

Comment:
Can you add `Expects to be called within the context of an instance of HlsHandler` to this. see `lastBandwidthSelector` jsdoc